experiments.json

```diff
@@ -61,6 +61,12 @@
             "desc": "<p>Test CSS transformations support in Servo.</p>",
             "href": "experiments/test-transformations/",
             "long_description": ""
+        },
+        {
+            "name": "WebGPU: Game of Life",
+            "desc": "<p>Test WebGPU support in Servo.</p>",
+            "href": "experiments/webgpu-game-of-life/",
+            "long_description": "<p>WebGPU demo built in <a href=\"https://codelabs.developers.google.com/your-first-webgpu-app\">Your first WebGPU app</a></p>"
```

Comment:
This link to `Your first WebGPU app` works fine, but it looks like text. Hence, as a user, I would never know it is a separate link.

<img width="383" alt="image" src="https://github.com/servo/servo-experiments/assets/94557773/67138cc5-7231-45c6-935a-e7003a231117">